README.md

```
@@ -63,70 +125,105 @@ For detailed changelog information, see [CHANGELOG.md](CHANGELOG.md).
+- ESLint with TypeScript and Airbnb base config
+- Strict TypeScript (`noUnusedLocals`,
+  `noUnusedParameters`, `noImplicitReturns`,
+  `noUncheckedIndexedAccess`)
+- Mocha + Chai + Sinon + esmock test framework
+- c8 V8-based code coverage
+- supertest for HTTP endpoint testing
+- CI runs on Node 20.x, 22.x, and 24.x
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)